src/common/dao/project.go

```diff
@@ -159,7 +160,7 @@ func ToggleProjectPublicity(projectID int64, publicity int) error {
 // which satisfies the following conditions:
 // 1. the project is not deleted
 // 2. the prject is public or the user is a member of the project
-func SearchProjects(userID int) ([]models.Project, error) {
+func SearchProjects(userID int) ([]*models.Project, error) {
```

Comment:
why return pointers here? Does "QueryRows" in orm supports both pointer to struct and struct?